lib/screens/filters_screen.dart

```diff
@@ -1,18 +1,96 @@
 import 'package:flutter/material.dart';
 import 'package:meals_app/widgets/main_drawer.dart';
 
-class FiltersScreen extends StatelessWidget {
+class FiltersScreen extends StatefulWidget {
   static const routeName = '/filters';
 
+  @override
+  _FiltersScreenState createState() => _FiltersScreenState();
+}
+
+class _FiltersScreenState extends State<FiltersScreen> {
+  var _glutenFree = false;
+  var _vegetarian = false;
+  var _vegan = false;
+  var _lactoseFree = false;
+
+  Widget _buildSwitchListTile(
+    String title,
+    String description,
+    var currentValue,
+    Function updateValue,
+  ) {
+    return SwitchListTile(
+      title: Text(title),
+      value: currentValue,
+      subtitle: Text(description),
+      onChanged: updateValue,
+    );
+  }
+
+  void changeGlutenFree(newValue) {
+    setState(() => _glutenFree = newValue);
+  }
+
+  void changeLactoseFree(newValue) {
+    setState(() => _lactoseFree = newValue);
+  }
+
+  void changeVegetarian(newValue) {
+    setState(() => _vegetarian = newValue);
+  }
+
+  void changeVegan(newValue) {
+    setState(() => _vegan = newValue);
+  }
```

Comment:
could you please pay attention parts like this next time? 😄  as I remembered in your previous pr I commented about making one-liner **_methods_**. 
![image](https://user-images.githubusercontent.com/41381713/102797972-640ed000-43eb-11eb-9899-93fc4365f96f.png)